atom/browser/resources/win/atom.manifest

```diff
@@ -32,7 +32,6 @@
 
   <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
     <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
-      <dpiAware>true</dpiAware>
```

Comment:
Non-client area scaling is available via the following:
1) <dpiAwareness>PerMonitor</dpiAwareness> tag + a call to EnableNonClinetDpiScaling() on 1607
2) <dpiAwareness>PerMonitorV2</dpiAwareness> on 1703 (no need for that additional API call)

You can also use a combination of tags that will be consumed differently on different versions of Windows:
<dpiAware>true/pm</dpiAware> (this will give you per-monitor (version 1) on >= 8.1 and system awareness on Vista to 8)
<dpiAwareness>PerMonitorV2, PerMonitor</dpiAwareness> will result in PerMonitorV2 on 1703 (Creators Update) and PerMonitor (version 1) on 1607 (Anniversary Update). When running on versions of Windows that consume this tag the <dpiAware> tag will be ignored.